{"event": "runner_on_failed", "uuid": "b8e0a357-102d-4bcb-bcc0-486c554ca76d", "stdout": "\u001b[0;31mfatal: [localhost]: FAILED! => {\"msg\": \"The task includes an option with an undefined variable. The error was: 'size' is undefined\\n\\nThe error appears to be in '/home/user/dev/cloudcat/creation.yml': line 5, column 7, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n  tasks:\\n    - debug:\\n      ^ here\\n\"}\u001b[0m", "counter": 16, "pid": 4619, "created": "2019-07-14T18:17:54.004201", "end_line": 16, "runner_ident": "ff3e9300-83cf-4935-9bd5-27ea67730901", "start_line": 15, "event_data": {"play_pattern": "localhost", "play": "localhost", "task": "debug", "task_args": "", "remote_addr": "127.0.0.1", "res": {"msg": "The task includes an option with an undefined variable. The error was: 'size' is undefined\n\nThe error appears to be in '/home/user/dev/cloudcat/creation.yml': line 5, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n  tasks:\n    - debug:\n      ^ here\n", "_ansible_no_log": false}, "pid": 4619, "play_uuid": "000c2955-ea58-f17e-f171-000000000005", "task_uuid": "000c2955-ea58-f17e-f171-000000000007", "event_loop": null, "playbook_uuid": "174a15e3-e110-45ea-8260-58b39596acbe", "playbook": "/home/user/dev/cloudcat/creation.yml", "task_action": "debug", "host": "localhost", "ignore_errors": null, "task_path": "/home/user/dev/cloudcat/creation.yml:5"}, "parent_uuid": "000c2955-ea58-f17e-f171-000000000007"}